storage/src/main/java/zipkin2/storage/kafka/KafkaStorage.java

```diff
@@ -109,7 +109,8 @@
         spansTopicName,
         autocompleteKeys,
         builder.traceGcInterval,
```

Comment:
sorry I mean figuratively like gc. I think the word gc will confuse with memory

@ImFlog @heymajo @jorgheymans @gquintana 

we are talking here about the interval to actually purge old traces after reaching a ttl. bikeshed painting help :)

traceCleanupInterval? traceReapInterval? wdyt?

Reply:
hahaha at least came up with `interval` :joy: 

I'm between `traceCleanupInterval` and `traceTtlCheckInterval` (found some `check.interval` configs here https://kafka.apache.org/documentation/#topicconfigs), do you have any preference?

Also, does `trace inactivity gap` is good name, or `trace alive  timeout` or something less attached to kafka stream would be better?

Reply:
:)

trace timeout seems clearest about what we are talking about (to me anyway). means regardless we flush a trace at this point.

good point about keeping same traceTtl prefix.. traceTtlInterval could even work. we don't need filler words. in any case you have to look to see what's happening on that interval. at least you know it is about the ttl.

Reply:
Right. Going for `trace-timeout` and `trace-ttl-check-interval` then :)